src/BubbleChart/BubbleChart.xml

```diff
@@ -76,6 +76,16 @@
                         <attributeType name="Integer"/>
                     </attributeTypes>
```

Comment:
Rename `Bubble size data attribute` to `>Bubble size attribute`

Reply:
Done.